src/ballet/ed25519/avx/fd_ed25519_fe.c

```diff
@@ -671,29 +671,29 @@ fd_ed25519_fe_mul121666( fd_ed25519_fe_t *       h,
   long h6 = 121666L * f6; long h7 = 121666L * f7;
   long h8 = 121666L * f8; long h9 = 121666L * f9;
 
-  long carry9 = (h9 + (1L << 24)) >> 25;
-  long carry1 = (h1 + (1L << 24)) >> 25;
-  long carry3 = (h3 + (1L << 24)) >> 25;
-  long carry5 = (h5 + (1L << 24)) >> 25;
-  long carry7 = (h7 + (1L << 24)) >> 25;
+  long carry9 = (h9 + fd_long_shift_left(1L, 24)) >> 25;
+  long carry1 = (h1 + fd_long_shift_left(1L, 24)) >> 25;
+  long carry3 = (h3 + fd_long_shift_left(1L, 24)) >> 25;
+  long carry5 = (h5 + fd_long_shift_left(1L, 24)) >> 25;
+  long carry7 = (h7 + fd_long_shift_left(1L, 24)) >> 25;
```

Comment:
Mostly indifferent to this but it doesn't matter to the UBSAN.  And the 1L << 25 didn't get the treatment below.  So, would probably keep this as (1L << 24).  Or in lieu of that, use the fortified shift_left on the 1<<25.